lib/temporal/execution_options.rb

```diff
@@ -14,6 +15,7 @@ def initialize(object, options, defaults = nil)
       @timeouts = options[:timeouts] || {}
       @headers = options[:headers] || {}
       @memo = options[:memo] || {}
+      @search_attributes = Temporal::Workflow::Context::Helpers.process_search_attributes(options[:search_attributes] || {}, allow_empty: true)
 
       # For Temporal::Workflow and Temporal::Activity use defined values as the next option
       if has_executable_concern?(object)
```

Comment:
I created the ability to specify default search attributes on the global `Temporal::Configuration` class, but I wasn't sure if I should also add the default to the Workflow class. This is because (if I understand it correctly) Workflow-class-defaults for the other properties (namespace, task queue, retry policy, timeouts, & headers) is done through the `Concerns::Executable` module/mixin, which is brought in for both activities and workflows (so adding a new option there for search attributes probably isn't appropriate).

Is this still something that should be implemented in this PR? And if so, how should it be done?

Reply:
Happy to postpone this change to a follow up PR to get this one out of the way first. As for the specific implementation — you're right that `Concerns::Executable` won't work here, but you can simply put it in the `Workflow` class since it is a behaviour specific to it

Reply:
Ok, let's leave that to a follow up PR, then

Reply:
Ok, let's leave that to a follow-up PR, then